demos/usage-animated-bfs.py

```diff
@@ -16,56 +16,56 @@
 app.css.config.serve_locally = True
```

Comment:
You can do this in a later PR if you want but just so we don't forget: all these `serve_locally` lines can be removed (has been the default since Dash v1.0!) and the imports (`html`, `dcc`, etc) should be [updated to Dash 2.x](https://dash.plotly.com/dash-2-0-migration#recommended-updates-%E2%80%94-import-statements)

Reply:
nonblocking, but the apps in the `demos` folder still have `serve_locally = True`

Reply:
My bad, I'll remove them